Makefile

```diff
@@ -29,12 +29,14 @@ default:
 	@echo "  build            - build docker containers"
 	@echo "  setup            - drop and recreate service state"
 	@echo "  run              - run webapp"
+	@echo "  runwebdebug      - run webapp outside gunicorn, for debugging"
```

Comment:
This comment is wrong. ``make run`` runs the app inside of ``waitress``, not ``gunicorn``.